.github/workflows/deploy.yml

```diff
@@ -1,68 +1,209 @@
+        run: |
+          container_id=$(docker create website)
+          docker cp ${container_id}:/opt/exercism/website/current/public/packs packs
+
+      - name: Upload assets to s3
+        run: |
+          aws s3 sync packs s3://${{ env.s3_bucket }}/packs/ --acl public-read --no-progress
+
+      #########################
+      #########################
+      # Build the Nginx image #
+      #########################
+      #########################
+      - name: Build and Push Nginx Docker
+        env:
+          ECR_REGISTRY: ${{ steps.login_to_ecr.outputs.registry }}
+          ECR_REPOSITORY: ${{ env.ecr_repository_nginx }}
+          IMAGE_TAG: ${{ github.sha }}
+          DOCKERFILE: ${{ env.dockerfile_nginx }}
+        run: |
+          # Build a docker container and push it to ECR
+          docker build -t $ECR_REGISTRY/$ECR_REPOSITORY:$IMAGE_TAG -f $DOCKERFILE .
+          docker push $ECR_REGISTRY/$ECR_REPOSITORY:$IMAGE_TAG
+
+          # Retag this as the latest
+          docker tag $ECR_REGISTRY/$ECR_REPOSITORY:$IMAGE_TAG $ECR_REGISTRY/$ECR_REPOSITORY:latest
+          docker push $ECR_REGISTRY/$ECR_REPOSITORY:latest
+
+          # Save the output for writing to the task definiton
+          echo "::set-output name=image::$ECR_REGISTRY/$ECR_REPOSITORY:$IMAGE_TAG"
+
+      ############################
+      ############################
+      # Build the AnyCable image #
+      ############################
+      ############################
+      - name: Build and Push Puma Docker
+        id: build_anycable_go
+        env:
+          ECR_REGISTRY: ${{ steps.login_to_ecr.outputs.registry }}
+          ECR_REPOSITORY: ${{ env.ecr_repository_anycable_go }}
+          IMAGE_TAG: ${{ github.sha }}
+          DOCKERFILE: ${{ env.dockerfile_anycable_go }}
+        run: |
+          # Build a docker container and push it to ECR
+          docker build -t $ECR_REGISTRY/$ECR_REPOSITORY:$IMAGE_TAG -f $DOCKERFILE .
+          docker push $ECR_REGISTRY/$ECR_REPOSITORY:$IMAGE_TAG
+
+          # Retag this as the latest
+          docker tag $ECR_REGISTRY/$ECR_REPOSITORY:$IMAGE_TAG $ECR_REGISTRY/$ECR_REPOSITORY:latest
+          docker push $ECR_REGISTRY/$ECR_REPOSITORY:latest
+
+          # Save the output for writing to the task definiton
+          echo "::set-output name=image::$ECR_REGISTRY/$ECR_REPOSITORY:$IMAGE_TAG"
+
+      ##############################
+      ##############################
+      # Update the task definition #
+      ##############################
+      ##############################
+
+      # We get the existing task definition to update
+      # Note that this is passed into the first update action
+      # but then chained from action to action for the others.
+      - name: Download task definition
+        run: |
+          aws ecs describe-task-definition --task-definition ${{ env.task_definition_name }} --query taskDefinition > task-definition.json
+
+      # Set puma to rails
```

Comment:
This (`puma`) and `anycable_rails` are two containers that use the same image, so we have two blocks to update in the file, although we've only created one image for both.

The task definition sets a custom ENTRYPOINT for the anycable one.